src/Field.tsx

```diff
@@ -78,6 +81,17 @@ export interface FieldConfig {
 
   /** Inner ref */
   innerRef?: (instance: any) => void;
+
+  shouldFieldUpdateOrEnableDefault?:
+    | boolean
+    | ((
+        props: FieldAttributes<any> & { formik: FormikContext<any> }
+      ) => boolean);
+
+  /** Should field need validate again */
+  shouldValidate?: (
+    meta: { values: any; value: any; fieldReason: string }
```

Comment:
what is `fieldReason`?

Reply:
`fieldReason` is the reason trigger validate, or the changing field